packages/@okta/i18n/src/properties/login.properties

```diff
@@ -811,7 +811,10 @@ oie.okta_verify.signed_nonce.label = Use Okta Verify on this device
 oie.okta_verify.enroll.force.upgrade.title = Update Okta Verify
 oie.authenticator.app.non_fips_compliant_enrollment_device_incompatible = The device used to set up Okta Verify does not meet your organization’s security requirements because it is not FIPS compliant. Contact your administrator for help.
 oie.authenticator.app.non_fips_compliant_enrollment_app_update_required = The Okta Verify version on the device used does not meet your organization’s security requirements. To add your account, update Okta Verify to the latest version, then try again.
-
+oie.authenticator.app.method.push.verify.enable.biometrics = We received your response, but your organization requires biometrics. To proceed, ensure that your device supports biometrics, then add your account and enable biometrics when prompted.
+oie.authenticator.app.method.push.verify.enable.biometrics.title = Enable biometrics in Okta Verify
+oie.authenticator.app.method.push.enroll.enable.biometrics = Your organization requires biometrics. To proceed, ensure your device supports biometrics, then add your account and enable biometrics when prompted.
```

Comment:
In this case does enrollment on client fail and users has to reenroll since UV required setting is in effect in authenticator settings ?

Reply:
Resoved over slack. The client app will show some generic error message.